ansible/roles/cleaner/handlers/main.yml

```diff
@@ -50,11 +50,13 @@
   systemd:
     name: soda-etcd
     state: stopped
+  when: ansible_distribution == "Ubuntu" and (ansible_lsb.major_release|int >=16 and ansible_lsb.major_release|int <=18)
```

Comment:
Instead of checking OS, can we check a variable to know if etcd is installed as systemd service?

Reply:
@joseph-v This is inside the handler. The handler code is invoked from the `ansible/roles/cleaner/tasks/main.yml` where the condition `install_as_systemd == true` is checked.